pkg/machinery/marker.go

```diff
@@ -69,3 +80,17 @@ type CodeFragments []string
 
 // CodeFragmentsMap binds Markers and CodeFragments together
 type CodeFragmentsMap map[Marker]CodeFragments
+
+func markerPrefix(prefix string) string {
+	trimmed := strings.TrimSpace(prefix)
```

Comment:
Will the trim revert the change:

> Bug fix to ensure consistent spacing in marker annotations (https://github.com/kubernetes-sigs/kubebuilder/pull/3904).

?

Reply:
@camilamacedo86 Please, correct my when I am wrong, but what I understand from the issue is, that it requests a constant string representation of the marker. This is still achieved by adding a space between the comment and the prefix:

```go
func (m Marker) String() string {
	return m.comment + " " + m.prefix + m.value
}
```

Reply:
Yes, you are right 👍